config.yml

```diff
@@ -37,6 +37,9 @@ redis_activity_consumers: 8
 # Number of concurrent connections to Redis in the connection pool.
 redis_connection_pool_size: 20
 
+# Max number of seconds BLPOP waits before returning anyway. Must be a positive integer. 0 means wait indefinitely.
+redis_blpop_timeout: 10
```

Comment:
why not wait indefinitely by default so when a message is there incus resolves it instantly? (unless this blocks redis somehow ofc)